### PR TITLE
[ISSUE-334] Pass NodeSelectors, LogLevel, ImageName

### PR DESCRIPTION
**Pull request by safronovD**
### Issue dell/csi-baremetal#334

Use parameters from CR Spec in resources creation:
- NodeSelectors
- LogLevel/Format
- ImageName/Tag/Repo 

## PR checklist
- [x] Add link to the issue
- [x] Choose PR label
- [ ] New unit tests added
- [ ] Modified code has meaningful comments
- [ ] All TODOs are linked with the issues
- [ ] All comments are resolved

## Testing
_Testing details_

### CODE REVIEW

**Review by AnastasiaShemyakinskaya — APPROVED** [review]

File: api/v1/components/log.go [hunk 1]
```diff
@@ -17,29 +17,29 @@ limitations under the License.
 package components
 
 // LogFormat is format in which log will appear
```
Comment (mishoyama) [inline api/v1/components/log.go:19]: Update comments as well.

File: config/samples/csi-baremetal_v1_deployment.yaml [hunk 2]
```diff
@@ -8,54 +8,57 @@ metadata:
...
+    #key: value
   driver:
     controller:
       image:
-        name: csi-baremetal-driver
-        pullPolicy: Always
-        tag: green
+        name: csi-baremetal-controller
+        pullPolicy: IfNotPresent
+        tag: 0.0.13-375.3c20841
```
Comment (mishoyama) [inline config/samples/csi-baremetal_v1_deployment.yaml:19]: Please leave `green` tag and `Always` pullPolicy by default.

**Review by mishoyama — APPROVED** [review]

File: api/v1/components/log.go (hunk 1 above)
Comment (safronovD) [reply api/v1/components/log.go]: was forgotten, fixed

File: config/samples/csi-baremetal_v1_deployment.yaml (hunk 2 above)
Comment (safronovD) [reply config/samples/csi-baremetal_v1_deployment.yaml]: reset

**Pull request merged**